olp-cpp-sdk-core/include/olp/core/client/BackdownStrategy.h

```diff
@@ -64,5 +64,54 @@ struct CORE_API ExponentialBackdownStrategy {
   }
 };
 
+/**
+ * @brief Computes wait time for the next retry attempt via
+ * the exponential backoff with the added jitter.
+ *
+ * This backoff strategy is based on the Equal Jitter approach. See
+ * https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/
```

Comment:
Please also add this link to the above ExponentialBackdownStrategy class Doxygen.